rancher/install.go

```diff
@@ -21,6 +21,61 @@ import (
 	"github.com/rancher-sandbox/ele-testhelpers/kubectl"
 )
 
+/** Support function for populating correct helm flags for Devel versions
+ * @param flags Helm flags
+ * @param headVersion Rancher head version
+ * @returns flags with correct values
+ */
+func appendDevelFlags(flags []string, headVersion string) []string {
+	if headVersion == "2.10" {
+		flags = append(flags,
+			"--devel",
+			"--set", "rancherImageTag=head",
+			"--set", "extraEnv[1].name=CATTLE_AGENT_IMAGE",
+			"--set", "extraEnv[1].value=rancher/rancher-agent:head",
+		)
+	} else if headVersion == "2.9" {
+		flags = append(flags,
+			"--devel",
+			"--set", "rancherImageTag=v"+headVersion+"-head",
+			"--set", "extraEnv[1].name=CATTLE_AGENT_IMAGE",
+			"--set", "extraEnv[1].value=rancher/rancher-agent:v"+headVersion+"-head",
+		)
+	} else {
+		// Devel images rancher:v2.7-head and rancher:v2.8-head are available on stgregistry.suse.com only
+		flags = append(flags,
+			"--devel",
+			"--set", "rancherImageTag=v"+headVersion+"-head",
+			"--set", "rancherImage=stgregistry.suse.com/rancher/rancher",
+			"--set", "extraEnv[1].name=CATTLE_AGENT_IMAGE",
+			"--set", "extraEnv[1].value=stgregistry.suse.com/rancher/rancher-agent:v"+headVersion+"-head",
+		)
+	}
+	return flags
+}
+
+/** Support function for populating correct helm flags for RC and Alpha versions
+ * @param flags Helm flags
+ * @param version Rancher version
+ * @param channel Rancher channel
+ * @returns flags with correct values
+ */
+func appendRCAlphaFlags(flags []string, version string, channel string) []string {
+	flags = append(flags,
+		"--devel",
```

Comment:
IMO `--devel` is useless here for versioned charts but I'll keeping it here as it should not hurt.

Reply:
Even versioned charts tagged as non-stable sometimes still require `--devel`, so yes I thin kit's better to keep it as you said. If you successfully test later then yes we can remove it in that case.